packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -73,22 +74,25 @@ impl DatabaseTestWrapper {
         }
     }
 
-    pub fn seed<D, P, E>(
+    pub fn seed<D, P, L, E>(
         &mut self,
         data_types: D,
         property_types: P,
+        link_types: L,
         entity_types: E,
     ) -> Result<
         (
             Vec<Qualified<DataType>>,
             Vec<Qualified<PropertyType>>,
+            Vec<Qualified<LinkType>>,
             Vec<Qualified<EntityType>>,
         ),
         InsertionError,
     >
```

Comment:
I think we don't use the return statement in any function. Maybe it's worth removing them? This would make the function a little bit more readable. I wonder if `clippy` does not run for the integration tests, I think this would trigger the type-complexity lint.

Reply:
Sure lets delete